db.go

```diff
@@ -63,3 +63,21 @@ func runMigrations(dsn string) error {
 	}
 	return err
 }
+
+// DropOldPartition drops number of days old partitions of pmm.metrics in ClickHouse.
+func DropOldPartition(db *sqlx.DB, days uint) {
+	partitions := []string{}
+	err := db.Select(
+		partitions,
+		`SELECT DISTINCT partition FROM system.parts WHERE partition < toYYYYMMDD(now() - toIntervalDay(?))`,
```

Comment:
Let's add `ORDER BY` for better logging below.

Reply:
@askomorokhov ^

Reply:
Added order by.